extras/signers.json

```diff
@@ -100,8 +100,6 @@
     "balancer_eoa": "0x7f87c1C42BeF332245F8B3cCAdD8224541CDaEcE"
   },
   "external-payees": {
-    "discord-antiscam-bot": "0x8c42138C925d1049EC6B29F1EcF817b1628e54Ba",
-    "drpc-usdc": "0x195A4B5583b2AB22C2a72Ef4E8f618adb0e29845",
-    "onchainification-labs": "0x94ac474B5Badc251Cf11C06713FB1E522de0747E"
+    "discord-antiscam-bot": "0x8c42138C925d1049EC6B29F1EcF817b1628e54Ba"
```

Comment:
Why is discord external but everything else just a payee?

Reply:
because it is an eoa (multichain)

Reply:
Do we know that this payment address accepts funds on all chains?  Is this an EOA someone controls or some coinbase payment wallet?

Reply:
it is clearly not a contract according to etherscan, and we can see it being active on at least five chains via debank: https://debank.com/profile/0x8c42138C925d1049EC6B29F1EcF817b1628e54Ba